examples/jsm/renderers/common/Renderer.js

```diff
@@ -533,70 +519,61 @@ class Renderer {
 
 	getContext() {
 
-		return this.backend.getContext();
+		return this.backend.getContext( this._defaultCanvasRenderTarget );
 
 	}
 
 	getPixelRatio() {
 
-		return this._pixelRatio;
+		return this._defaultCanvasRenderTarget.pixelRatio;
 
 	}
 
-	getDrawingBufferSize( target ) {
+	getActivePixelRatio() {
```

Comment:
Is it necessary to introduce all the `getActive*()` methods? Could `getPixelRatio()` just work like so:

- Report the pixel ratio of the default canvas render target.
- If a custom canvas render target is set, return its values instead.

Reply:
I considered that but was worried about altering the behaviour of those methods, happy to do that instead though, or add a method to getActiveCanvasTarget()  for use within the Node classes.